code/languages/com.mbeddr.mpsutil/languages/com.mbeddr.mpsutil.actionsfilter/languageModels/editor.mps

```diff
@@ -439,11 +441,27 @@
         </node>
       </node>
       <node concept="3F0ifn" id="7LkwBl9HKH4" role="3EZMnx" />
-      <node concept="3F0ifn" id="6nrtUqYcU5t" role="3EZMnx">
-        <property role="3F0ifm" value="specify toolbar" />
-        <node concept="VechU" id="6nrtUqYcU5u" role="3F10Kt">
-          <property role="Vb096" value="DARK_BLUE" />
+      <node concept="3EZMnI" id="7I52n9WCCw9" role="3EZMnx">
+        <node concept="VPM3Z" id="7I52n9WCCwc" role="3F10Kt">
+          <property role="VOm3f" value="false" />
+        </node>
+        <node concept="3F0ifn" id="7I52n9WCCwe" role="3EZMnx">
+          <property role="3F0ifm" value="specify toolbar" />
+          <node concept="VechU" id="7I52n9WCCwf" role="3F10Kt">
+            <property role="Vb096" value="DARK_BLUE" />
+          </node>
+        </node>
+        <node concept="3F0ifn" id="6hyuZyX9wzG" role="3EZMnx" />
+        <node concept="3F0ifn" id="7I52n9WCDhs" role="3EZMnx">
+          <property role="3F0ifm" value="Toolbar actions cannot be filtered out from the whole action profile" />
```

Comment:
Can we change this to something like `Make sure actions from the toolbar aren't filtered elsewhere` and maybe also mention that it would cause problems at runtime otherwise?